tutorials/scripting/gdextension/gdextension_docs_generation.rst

```diff
@@ -0,0 +1,47 @@
+.. _doc_gdextension_docs_generation:
+
+Adding documentation to your GDExtension
+========================================
```

Comment:
I suggest making the page title more factual. We generally try not to refer to the reader in page titles (with a few exceptions like *Your first game*):

```suggestion
GDExtension documentation system
================================
```